src/libostree/ostree-fetcher-curl.c

```diff
@@ -329,7 +329,11 @@ check_multi_info (OstreeFetcher *fetcher)
             }
           else
             {
-              g_task_return_new_error (task, G_IO_ERROR, G_IO_ERROR_FAILED,
+              /* When it is not a file, we want to retry the request.
+               * We accomplish that by using G_IO_ERROR_TIMED_OUT.
+               */
+              int g_io_error_code = is_file ? G_IO_ERROR_FAILED : G_IO_ERROR_TIMED_OUT
```

Comment:
Need a `;` at the end here

Reply:
man, not passing the ` --with-curl` flag got me again. fixed.

Also it builds OK with:
```
env CFLAGS='-ggdb -Og' CXXFLAGS='-ggdb -Og' ./autogen.sh \ 
    --disable-silent-rules --enable-gtk-doc --with-selinux --with-curl \ 
    --with-openssl --with-dracut=yesbutnoconf --prefix=/usr \
    --libdir=/usr/lib64 --sysconfdir=/etc && make
```

Will get it out of draft as soon as I test this actually works.